### DISCUSSION TITLE
Feedback for “Frequently Asked Questions”

### DISCUSSION THREAD

**Discussion by CesarRAsa** [callstack/react-native-builder-bob#741] [Q&A] [answered]
Quick question, guys if we were going to try to use a mini application inside of an application that includes basic permissions, ui and sometimes navigation, since we do not have to deal with native code due to the fact that we want the client app to actually configure that , just to install the right dependencies in peerDependencies will suffice right? or do we still need to create a Turbomodule/Fabric? what will be the right approach here?

**Comment by atlj** [comment, marked as answer]
Hey, you can pick the `JavaScript` only library option in this case. You won't need any native logic. And as you said, `peerDependencies` should be enough. Just make sure to also put them in the `devDependencies` as well for typescript. As long as no React Native dependency is listed in the `dependencies`, it should be fine.

**Reply by CesarRAsa** [reply]
Thanks for your help, very much appreciate it!
